src/graphql/root/mutation/intraledger-payment-send.ts

```diff
@@ -65,7 +55,8 @@ const IntraLedgerPaymentSendMutation = GT.Field({
       memo,
       amount,
       senderWalletId: walletId,
-      senderAccount: domainAccount,
+      // FIXME: domainAccount should not be undefined if we reach this stage. Improve type
```

Comment:
Looks like another unintentional reversion

Reply:
it is intentional. just the result of the same PR (we need to improve the types at the graphql layer)

Reply:
basically we know the domainAccount is not undefined here, but typescript doesn't know it.

Reply:
ok I have understood what to do to remove the type error